tests/system/runtimes/test_nuclio.py

```diff
@@ -91,6 +91,43 @@ def test_workaround_for_nuclio_bug(self):
             assert resp.status_code == 200
             assert resp.text == "NoneType"
 
+    def test_nuclio_function_status_fields(self):
+        code_path = str(self.assets_path / "nuclio_function_to_print_type.py")
+
+        self._logger.debug("Creating nuclio function")
+        function = mlrun.code_to_function(
+            name="test-function",
+            kind="serving",
+            project=self.project_name,
+            filename=code_path,
+            image="mlrun/mlrun",
+        )
+
+        graph = function.set_topology("flow", engine="sync")
```

Comment:
Is this related to the test? looks like it can be removed

Reply:
Since I'm deploying a serving function in the test, it must have a graph topology.